video_xblock/video_xblock.py

```diff
@@ -749,6 +761,56 @@ def update_metadata_authentication(self, auth_data, player):
             self.metadata['client_id'] = ''  # Brightcove API
             self.metadata['client_secret'] = ''  # Brightcove API
 
+    @XBlock.handler
+    def upload_manual_transcript_handler(self, request, suffix=''):
+        """
+        Upload manual transcript for video xblock.
+
+        Saves transcript as the .vtt file.
+        Performs file conversion beforehand (if a different supported file type is received).
+        
+        Arguments:
+            request: Data from frontend on a transcript.
+            _suffix: Slug used for routing. Imposed by `XBlock.handler`.
+        Returns: Transcript data uploaded manually.
+        """
+        upload = request.params.get("file")
+        upload_file = upload.file
+        course_key_string = request.params.get('course_key')
+        course_key = CourseKey.from_string(course_key_string)
+        filename = upload_file.name.replace(" ", "_")
+        
+        # Files in the .vtt format are saved without conversion. 
+        # The transcript in the .srt format should be converted to the .vtt format 
+        # and then saved using the standard flow.
+        if not filename.endswith('.vtt'):
+            upload_file = self._convert_file_to_vtt(upload_file, filename)
+            if not upload_file:
+                return Response(
+                    json.dumps({'message': _("Failed to convert file.")}),
+                    status_code=422,
+                    content_type='application/json',
+                    charset='utf8'
```

Comment:
I saw several similar use cases for such an argument across the platform (e.g. `charset='utf8'`, `charset='utf-8'`, `charset='UTF-8'`)

I'm curious if they are interchangeable

Reply:
n the context of HTTP responses, there is no difference between using charset='utf8' and charset='utf-8', as both variants refer to the same character encoding.